internal/scorecard/alpha/scorecard.go

```diff
@@ -99,3 +122,30 @@ func ConfigDocLink() string {
 		"https://github.com/operator-framework/operator-sdk/blob/%s/doc/test-framework/scorecard.md",
 		version.Version)
 }
+
+// waitForTestsToComplete waits for a fixed amount of time while
+// checking for test pods to complete
+func waitForTestsToComplete(o Options, tests []ScorecardTest) (err error) {
+	for elapsedSeconds := 0; elapsedSeconds < o.WaitTime; elapsedSeconds++ {
+		allPodsCompleted := true
+		for i := 0; i < len(tests); i++ {
+			p := tests[i].TestPod
+			var tmp *v1.Pod
+			tmp, err = o.Client.CoreV1().Pods(p.Namespace).Get(p.Name, metav1.GetOptions{})
+			if err != nil {
+				fmt.Printf("error getting pod %s %s\n", p.Name, err.Error())
```

Comment:
why is required to print? Will not the error be returned? So, will not it be outputted again where it throws?

Reply:
fixed it to return just an error.